fastlane_bot/helpers/txhelpers.py

```diff
@@ -991,19 +991,22 @@ def _get_layer_one_gas_fee_loop(self, signed_transaction) -> Decimal:
         returns: Decimal
             The total fee (in gas token) for the l1 gas fee
         """
-        return asyncio.get_event_loop().run_until_complete(self._get_layer_one_gas_fee(signed_transaction=signed_transaction))
 
-    async def _get_layer_one_gas_fee(self, signed_transaction) -> Decimal:
+        tasks = self.ConfigObj.GAS_ORACLE_CONTRACT.caller.basefee(), self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeOverhead(), self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeScalar()
+        ethereum_base_fee, fixed_overhead, dynamic_overhead = asyncio.get_event_loop().run_until_complete(asyncio.gather(*tasks))
```

Comment:
Cleaner IMO (instead of creating an array of tasks and then dereferencing it with `*`):
```
ethereum_base_fee, fixed_overhead, dynamic_overhead = asyncio.get_event_loop().run_until_complete(
    asyncio.gather([
        self.ConfigObj.GAS_ORACLE_CONTRACT.caller.basefee(),
        self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeOverhead(),
        self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeScalar()
    ])
)
```
Also, please search if there's a function which achieves what `run_until_complete(asyncio.gather(...))`.
Make sense that there should be a single function which takes care of that.

Reply:
Updated this.